### DISCUSSION TITLE
Visualization of hydrophobic interactions in 2D/3D

### DISCUSSION THREAD

**Discussion by ONISniperOne** [chemosim-lab/ProLIF#177] [Ideas]
Was just wondering if you can add an option to toggle or change the 2D/3D depiction of hydrophobic interactions from a pseudobond to eyelashes in 2D or a surface mesh in 3D? Kinda like ligplot depiction.

**Comment by cbouy** [comment]
That would be quite complicated with the backend used by the `LigNetwork` plot so probably never gonna happen, but I might (someday) have a go at using `InteractionDrawer`, the open-source library behind [this](https://link.springer.com/article/10.1007/s10822-023-00522-4/figures/4) sort of plots. But I haven't looked too much into the details yet, so probably not going to happen soon

**Comment by ONISniperOne** [comment]
> That would be quite complicated with the backend used by the `LigNetwork` plot so probably never gonna happen, but I might (someday) have a go at using `InteractionDrawer`, the open-source library behind [this](https://link.springer.com/article/10.1007/s10822-023-00522-4/figures/4) sort of plots. But I haven't looked too much into the details yet, so probably not going to happen soon

Thank you!
